perpVects/QFPHelpers.h

```diff
@@ -92,6 +97,30 @@ namespace FPHelpers {
     return val & (~zero >> 48);
   }
 
+  // template <typename T>
+  // int
+  // swap_float_int(T val){
+  //   return *reinterpret_cast<int*>(&val);
+  // }
```

Comment:
Can this be deleted?  Or is it here to say "this is what the following functions are achieving"?